Archipelago.HollowKnight/Archipelago.HollowKnight.csproj

```diff
@@ -114,218 +111,212 @@
     <Reference Include="System.IO.Compression" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Runtime.Serialization.Xml" />
-    <Reference Include="System.ServiceModel.Internals">
-      <HintPath>..\DLLs\System.ServiceModel.Internals.dll</HintPath>
-    </Reference>
     <Reference Include="System.Transactions">
       <HintPath>..\DLLs\System.Transactions.dll</HintPath>
     </Reference>
     <Reference Include="System.Xml.XPath.XDocument">
       <HintPath>..\DLLs\System.Xml.XPath.XDocument.dll</HintPath>
     </Reference>
```

Comment:
These appear to have been missed. Can you remove these references if they aren't necessary?